roles/ansible-runner/tasks/main.yml

```diff
@@ -74,13 +81,6 @@
     dest: "/opt/source/{{ ansible_runner_job }}"
     update: true
 
-- name: Set source directory owner
-  file:
-    state: directory
-    path: "/opt/source/{{ ansible_runner_job }}"
-    recurse: yes
-    owner: "{{ ansible_runner_user }}"
-
```

Comment:
This should be fine to remove because all the files are +r for other, and cideploy can read them.